production/helm/values.yaml

```diff
@@ -97,6 +97,8 @@ promtail:
   nameOverride: promtail
   deploymentStrategy: RollingUpdate
 
+  dataRoot: /var/lib/docker
```

Comment:
This implies the container runtime is docker, which isn't always the case. I'm using containerd and it stores data under /var/lib/containerd and there is no containers sub-directory. It would be better to customize the entire hostPath and not assume there is a /containers.

Reply:
I believe the cri spec says logs must be located at /var/log/containers. This can be symlinked wherever the runtime decides it should go. So maybe skip this and have an extraMounts section?

Reply:
There is a different mount already for /var/log and that looks to be fine, this is about /var/lib/docker.

Reply:
I think that this PR give uses an example about custom data root, support all cri may be a new PR @steven-sheehy

Reply:
extraMounts would let you to add in /var/lib/docker/ or /var/lib/containerd or whatever else needs to go in.